.github/workflows/ci.yml

```diff
@@ -377,13 +382,20 @@ jobs:
           mv ./clash-rs/clash-x86_64-unknown-linux-musl ./clash-rs/clash-amd64
           mv ./clash-rs/clash-aarch64-unknown-linux-musl ./clash-rs/clash-arm64
 
+      - name: Log in to Container Registry
```

Comment:
is this intentional?

Reply:
Yes，I about to refactor docker workflow，but failed. Probably another pr